coding-concepts.js

```diff
@@ -7,48 +7,48 @@
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
 const colors = ["tangerine", "magenta", "lilac", "daffodil"]
-// console.log(colors.push("indigo"))
+ console.log(colors.push("indigo"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: This will put the string "indigo" into the variable colors.  When logged, it will show 5 showing that the array called colors now has 5 strings inserted instead of 4.
+// b) Verify and explain: When I logged this into the terminal with the node command it showed 5 confirming what I thought.  Again, this is just showing us how many data values are now in the array after it pushes the string "indigo" into it.
 
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2023"
-// console.log(cohort.length)
+console.log(cohort.length)
 
-// a) Your answer:
+// a) Your answer: This will show us the length of the string "LEARN 2023".
 // b) Verify and explain:
-
+// I logged the method and it came back as 10. 10 is the amount of characters in the string including the quotations on either side.
```

Comment:
It does not count the quotations in the length of the string. However, it does count the space as a character.